graylog2-server/src/main/java/org/graylog2/indexer/MongoIndexSet.java

```diff
@@ -63,6 +63,7 @@ public class MongoIndexSet implements IndexSet {
     // TODO 3.0: Remove this in 3.0, only used for pre 2.2 backwards compatibility.
     public static final String RESTORED_ARCHIVE_SUFFIX = "_restored_archive";
     public static final String WARM_INDEX_INFIX = "warm_";
+    private static final String WARM_INDE_INFIX_WITH_SEPERATOR = SEPARATOR + WARM_INDEX_INFIX;
```

Comment:
Didn't look at the PR but just stumbled upon a double typo in the field name while browsing.

```suggestion
    private static final String WARM_INDEX_INFIX_WITH_SEPARATOR = SEPARATOR + WARM_INDEX_INFIX;
```

Reply:
thanks